modules/ROOT/pages/alerts-config.adoc

```diff
@@ -5,17 +5,61 @@ endif::[]
 
 Depending on your subscription level, you can configure different types of alerts:
 
-* Operational alerts for APIs through API Manager
-* Operational alerts for the runtime through Runtime Manager
-* Alerts for graphs in custom dashboards in Anypoint Monitoring.
+* Basic alerts for servers and Mule apps
 +
-Limit: Up to 20 alerts per organization.
+Limit per organization: Up to 50 basic alerts for users who do not have a Titanium subscription to Anypoint Platform. If you have a Titanium subscription, the limit is 100 basic alerts, and the limits increase as you purchase more vCores. For complete details about subscriptions, contact your customer support representative. See also https://www.mulesoft.com/anypoint-pricing[Anypoint Platform pricing].
+* Advanced alerts for graphs in custom dashboards in Anypoint Monitoring
 +
-Note that thresholds are checked every 5 minutes. The evaluated query is the average value of the metric in the last 5 minutes. The evaluated metric is the one referenced in the General tab for the graph. If the value is above the threshold, the alert is triggered. The alert is triggered only if the alert state changes (from OK state to Alerting state or vice versa).
+Limit per organization: Up to 20 advanced alerts.
++
+Note that thresholds are checked every 5 minutes. The evaluated query is the average value of the metric in the last 5 minutes. The evaluated metric is the one referenced in the *General* tab for the graph. If the value passes the threshold, the advanced alert is triggered. The advanced alert is triggered only if the alert state changes (from `OK` state to `Alerting` state or vice versa).
+* Operational alerts for APIs through API Manager
+* Operational alerts for the runtime through Runtime Manager
+
+[[basic-alerts]]
+== Set up Basic Alerts for Servers and Mule Apps
+
+You can set up basic alerts to trigger email notifications when a metric you are measuring passes a specified threshold.
```

Comment:
Ah! "email notifications" is better than "notifications to email" earlier....

Reply:
Implementing this everywhere. THANK YOU. So much more concise...